### PR TITLE
Fixed Some Routing (WG) and Some Optimization

### PR DESCRIPTION
**Pull request by xAJx1383**
radarroute.json routes wasn't applied ...

### CODE REVIEW

File: RadarGame/RadarGame.UI/Services/VpnNetworkManager.cs [hunk 1]
```diff
@@ -25,12 +25,13 @@ public VpnNetworkManager(IConnectionObserver connectionObserver)
...
-            timer = new Timer(checkConnection, null, TimeSpan.FromSeconds(5), TimeSpan.FromSeconds(5));
+            checkConnection( null);
+            //timer = new Timer(, TimeSpan.FromSeconds(5), TimeSpan.FromSeconds(5));
         }
 
         public void StopVpnConnectionObserver()
         {
-            timer?.Change(Timeout.Infinite, Timeout.Infinite);
+            //timer?.Change(Timeout.Infinite, Timeout.Infinite);
             if (timer != null)
```
Comment (ZenMaxe) [inline RadarGame/RadarGame.UI/Services/VpnNetworkManager.cs:35]: We Have Timer Here Check The If

**Review by ZenMaxe — COMMENTED** [review]
Performance Issue With EmptyWorkingSet

File: RadarGame/RadarGame.UI/Services/VpnNetworkManager.cs (hunk 1 above)
Comment (xAJx1383) [reply RadarGame/RadarGame.UI/Services/VpnNetworkManager.cs:35]: i didint want to use timer because of performance penalty

**Comment by xAJx1383** [comment]
stutter in UI or High usage of CPU?
it might stutter in UI while its connecting if you notice every thing else tell me if you like to

File: RadarGame/RadarGame.UI/Services/VpnNetworkManager.cs (hunk 1 above)
Comment (xAJx1383) [reply RadarGame/RadarGame.UI/Services/VpnNetworkManager.cs:35]: i think , in v1.0.0.1 there is a check for network interface if pinging timed out...

**Pull request closed without merging**